pkg/pdp/aggregator/jobqueue/internal/testing/testing.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) https://github.com/maragudk/goqite
+// https://github.com/maragudk/goqite/blob/6d1bf3c0bcab5a683e0bc7a82a4c76ceac1bbe3f/LICENSE
+//
+// This source code is licensed under the MIT license found in the LICENSE file
+// in the root directory of this source tree, or at:
+// https://opensource.org/licenses/MIT
+
+package testing
+
+import (
+	"database/sql"
+	_ "embed"
+	"fmt"
+	"testing"
+
+	_ "github.com/ncruces/go-sqlite3"
```

Comment:
curious how you picked this sqlite driver. there are so many. mostly curious cause I need to pick one for the upload client

Reply:
Primary motivation was it's pure go and thus cgo-free - really would like to avoid cgo in this repo. 

I reviewed the options outlined here: https://github.com/cvilsmeier/go-sqlite-bench and this package was one of the best performers in the pure-go category.